src/extend/withPlugins.js

```diff
@@ -48,7 +49,7 @@ function _withPlugins(targetName, TargetComponent) { // eslint-disable-line no-u
   const whatever = React.forwardRef(PluginHoc);
 
   whatever.displayName = `WithPlugins(${targetName})`;
-  return whatever;
+  return withTranslation()(whatever);
```

Comment:
I'm not sure if this is something bad here.. getting some warnings about `t` in the props validation. Would like a second set of eyes.

Reply:
Couldn't plugins call withTranslation themselves? Or is this another weird JS wiring issue?

Reply:
Just verified.. no they totally can. I'm going to pop off this commit.